help/text-emphasis.md

```diff
@@ -0,0 +1,45 @@
+# Text emphasis
+
+Zulip supports Markdown syntax for bold, italic, and strikethrough text.
+
```

Comment:
Can we use "Markdown formatting" rather than "Markdown syntax" across the board?